Safety/AttitudeManager/Src/attitudeStateClasses.cpp

```diff
@@ -65,7 +66,9 @@ void fetchInstructionsMode::execute(attitudeManager* attitudeMgr)
     //TODO: Determine if RC is commanding to go autonomous
     bool isTeleopCommandingAuto = false;
     
-    if(!isTeleopCommandingAuto || GetFromPMToAM(&_PMInstructions))
+    // If drone receives instructions from RC indicating autonomous mode or instructions from PM,
+    // then it is in autonomous mode
```

Comment:
Not sure about this comment since this part doesn't actually determine if it is in autonomous mode. I would describe the if-else like: "PM will timeout if it fails to report data while autonomous mode is triggered"